doc/man3/SSL_get_event_timeout.pod

```diff
@@ -0,0 +1,87 @@
+=pod
+
+=head1 NAME
+
+SSL_get_event_timeout - determine when an SSL object next needs to be ticked
```

Comment:
Rephrase "needs to be ticked", since we're not using the tick terminology any more.